sections/main-cart-items.liquid

```diff
@@ -125,6 +125,17 @@
                         </div>
                       {%- endif -%}
 
+                      {%- for component in item.item_components -%}
+                      <div class="product-option">
+                        <sub>
+                          {{ component.title }}
+                          {%- if component.quantity > 1 -%}
+                            <span> × {{ component.quantity }}</span>
+                          {%- endif -%}
+                        </sub>
```

Comment:
I wonder if this is the right element for the job. Id go with an unordered list instead maybe.

When i look at the sub element in docs, MDN for example, it does mention using them in specific scenarios: 
>should be displayed as subscript for solely typographical reasons